### ISSUE TITLE
Jest based testing is broken

### ISSUE THREAD

**Issue by tonyxiao** [fbsamples/f8app#52]
Is anybody else encountering this problem?

Environment
```
Tony:f8app (master)$ git status
On branch master
Tony:f8app (master)$ git log -n 1
commit 78233d56f404ea284999d51b37e5e8ae11f4063d
Author: Alexander Kotliarskyi <alex.frantic@gmail.com>
Date:   Thu May 5 11:25:22 2016 -0700

    Fix #35
Your branch is up-to-date with 'origin/master'.
nothing to commit, working directory clean
Tony:f8app (master)$ npm -v
3.8.6
Tony:f8app (master)$ node -v
v6.0.0
```
Error
```
Tony:f8app (master)$ npm test

> F8v2@0.0.1 test /Users/Tony/Code/tonyxiao/f8app
> jest

Using Jest CLI v0.9.2, jasmine2, babel-jest
 FAIL  js/tabs/schedule/__tests__/formatTime-test.js
● Runtime Error
TypeError: Path must be a string. Received null
    at assertPath (path.js:7:11)
    at Object.dirname (path.js:1324:5)
    at handle (/Users/Tony/Code/tonyxiao/f8app/node_modules/worker-farm/lib/child/index.js:41:8)
    at process.<anonymous> (/Users/Tony/Code/tonyxiao/f8app/node_modules/worker-farm/lib/child/index.js:47:3)
 FAIL  js/reducers/__tests__/maps-test.js
● Runtime Error
TypeError: Path must be a string. Received null
    at assertPath (path.js:7:11)
    at Object.dirname (path.js:1324:5)
    at handle (/Users/Tony/Code/tonyxiao/f8app/node_modules/worker-farm/lib/child/index.js:41:8)
    at process.<anonymous> (/Users/Tony/Code/tonyxiao/f8app/node_modules/worker-farm/lib/child/index.js:47:3)
 FAIL  js/reducers/__tests__/schedule-test.js
● Runtime Error
TypeError: Path must be a string. Received null
    at assertPath (path.js:7:11)
    at Object.dirname (path.js:1324:5)
    at handle (/Users/Tony/Code/tonyxiao/f8app/node_modules/worker-farm/lib/child/index.js:41:8)
    at process.<anonymous> (/Users/Tony/Code/tonyxiao/f8app/node_modules/worker-farm/lib/child/index.js:47:3)
 FAIL  js/tabs/schedule/__tests__/formatDuration-test.js
● Runtime Error
TypeError: Path must be a string. Received null
    at assertPath (path.js:7:11)
    at Object.dirname (path.js:1324:5)
    at handle (/Users/Tony/Code/tonyxiao/f8app/node_modules/worker-farm/lib/child/index.js:41:8)
    at process.<anonymous> (/Users/Tony/Code/tonyxiao/f8app/node_modules/worker-farm/lib/child/index.js:47:3)
 FAIL  js/reducers/__tests__/notifications-test.js
● Runtime Error
TypeError: Path must be a string. Received null
    at assertPath (path.js:7:11)
    at Object.dirname (path.js:1324:5)
    at handle (/Users/Tony/Code/tonyxiao/f8app/node_modules/worker-farm/lib/child/index.js:41:8)
    at process.<anonymous> (/Users/Tony/Code/tonyxiao/f8app/node_modules/worker-farm/lib/child/index.js:47:3)
npm ERR! Test failed.  See above for more details.
```

**Comment by StephanBijzitter** [comment]
+1 same stacktrace

**Comment by tonyxiao** [comment]
Update, tests can be fixed by updating to `jest-cli` `12.1.1`

**Comment by tonyxiao** [comment]
Closing this issue, please see pull request.
